### DISCUSSION TITLE
기능 요구사항이 일부 변경 안내

### DISCUSSION THREAD

**Discussion by gracefulBrown** [woowacourse/java-subway-map-precourse#1] [General]
웹 백엔드 프리코스 3주차 미션의 기능 요구사항이 일부 변경되어 알려드립니다.

## 지하철 역 관련 기능 
- 지하철 역을 등록하고 삭제할 수 있다. (단, 노선에 등록된 역은 삭제할 수 없다) 
- 중복된 지하철 역 이름이 등록될 수 없다. 
- ~역 이름은 2자 이상, 5자 이하만 가능하다.~  -> 지하철 역 이름은 2글자 이상이어야 한다. 
- 지하철 역의 목록을 조회할 수 있다. 

## 지하철 노선 관련 기능 
- 지하철 노선을 등록하고 삭제할 수 있다. 
- 중복된 지하철 노선 이름이 등록될 수 없다. 
- ~노선 이름은 2자 이상, 5자 이하만 가능하다.~ -> 지하철 노선 이름은 2글자 이상이어야 한다. 
- 노선 등록 시 상행 종점역과 하행 종점역을 입력받는다. 
- 지하철 노선의 목록을 조회할 수 있다.

감사합니다.

**Comment by Livenow14** [comment]
확인했습니다. 감사합니다.

**Comment by pkeugine** [comment]
🙌 확인했습니다!
